scm/src/main/java/com/sandboni/core/scm/scope/ChangeScopeImpl.java

```diff
@@ -81,7 +85,9 @@ public void remove(FileExtensions... fe) {
 
     @Override
     public void include(FileExtensions... fe) {
+        logger.debug("Start ChangeScopeImpl:include");
```

Comment:
Is this temporary? 
and - don't you want to add the file extensions info?

Reply:
Well is at debug level, it will be printed out only if debug is enabled. File extensions are not relevant so I decided to skip them.